cmake/dependencies/curlpp.cmake

```diff
@@ -0,0 +1,73 @@
+##########
+# cURLpp #
```

Comment:
You should add them to the dependencies filter within visual studio, see here:

https://github.com/FLAMEGPU/FLAMEGPU2/blob/4259623ef2df930ef8f9057b9c08a78df818c560/src/CMakeLists.txt#L508